analyzer/topology_publisher_endpoint.go

```diff
@@ -66,12 +66,11 @@ func (t *TopologyPublisherEndpoint) OnDisconnected(c shttp.WSSpeaker) {
 		return
 	}
 
-	host := c.GetRemoteHost()
-
-	logging.GetLogger().Debugf("Authoritative client unregistered, delete resources %s", host)
+	origin := string(c.GetServiceType()) + "." + c.GetRemoteHost()
```

Comment:
We should set the type to something in the python api if not provided, something like "skydive-python-client"
https://github.com/skydive-project/skydive/blob/master/contrib/python/api/skydive/websocket/client.py#L146

Reply:
Sorry Masco but I think we need to work a bit more on this. We need to remove the origin field from the python client library because a client should not be able to specify the origin otherwise he could set it to something different than "type + host" then not being able to DelOriginGraph.

So I propose to force type to be not empty in the python client (meaning adding a check) and to add something in the 
https://github.com/skydive-project/skydive/blob/master/analyzer/topology_publisher_endpoint.go#L80
to set the origin with the GetServiceType function. So that we guaranty that the node/edge created by a client has the correct origin.